lib/client-assets.php

```diff
@@ -186,6 +193,10 @@ function gutenberg_register_scripts_and_styles() {
 		'wp-dom',
 		gutenberg_get_script_polyfill( array(
 			'document.contains' => 'wp-polyfill-node-contains',
+			'"DOMRect" in this && ( function( DOMRect ) {' .
+				'try { return new DOMRect(); }' .
```

Comment:
This relies on `gutenberg_get_script_polyfill` interpreting the result of the test as truthy, not explicitly `true`, since this will return an instance of `DOMRect`. While it may work as-is, I don't think it's a very strong guarantee. I might suggest simply changing this to `new DOMRect(); return true;`